app/src/sharedTest/java/org/oppia/android/app/walkthrough/WalkthroughTopicListFragmentTest.kt

```diff
@@ -151,7 +151,7 @@ class WalkthroughTopicListFragmentTest {
   }
 
   @Test
-  fun testWalkthroughWelcomeFragment_recyclerViewIndex1_topicCard_configurationChanged_topicNameIsCorrect() { // ktlint-disable max-line-length
+  fun testWalkthroughWelcomeFrag_recyclerViewIndex1_topicCard_configChanged_topicNameIsCorrect() {
```

Comment:
```kotlin
fun testWalkthroughTopicListFragment_topicCard_configurationChanged_topicNameIsCorrect()
```

please also correct other function names too as it's`testWalkthroughTopicListFragment`

Reply:
should I remove the name of the fragment from the function name? since the file already specifies the name of the fragment. also, this would make all function name under 100 chars.

Reply:
this is something mentioned in the naming convention which we have to follow- https://github.com/oppia/oppia-android/wiki/Oppia-Android-Testing#naming-convention

Although you can file a new issue for this and suggest a better naming convention and also I feel in that way we will never have this 100 char issue.

Reply:
I thought of naming convention like `test_withOneCondition_withSecondCondition_hasExpectedOutcome` but this can create problems between tests like `testEmptyStringAns_emptyStringInput_exactSameStrings_verifyAnsStartsWith` and `testUppercaseStringAns_emptyStringInput_differentStrings_verifyAnsStartWith` @anandwana001 WDYT

Reply:
File a new issue for discussion